sql/core/src/test/scala/org/apache/spark/sql/DatetimeExpressionsSuite.scala

```diff
@@ -45,4 +48,138 @@ class DatetimeExpressionsSuite extends QueryTest {
       0).getTime - System.currentTimeMillis()) < 5000)
   }
 
+
+
+  val sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
+  val sdfDate = new SimpleDateFormat("yyyy-MM-dd")
+  val d = new Date(sdf.parse("2015-04-08 13:10:15").getTime)
+  val ts = new Timestamp(sdf.parse("2013-04-08 13:10:15").getTime)
+
+
+  test("date format") {
```

Comment:
since we have a lot of date functions, do you mind taking all the date related tests out into a separate file?